docs/tutorials/first-instance.md

```
@@ -0,0 +1,228 @@
+
+## Step 6 — Watch it come up
+
+```sh
+kubectl get ha home -w
+```
+
+This takes two to three minutes on a first run, because the Home Assistant image
+has to be pulled and Home Assistant needs to initialise its database. Press
+++ctrl+c++ to stop watching once you see:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)